src/index.ts

```diff
@@ -1,13 +1,20 @@
 import express, { Request, Response, Express } from 'express';
 import swaggerUi from 'swagger-ui-express';
+import mongoose from 'mongoose';
 
 import swaggerSpec from './utils/swagger';
 import errorHandler from './middleware/error-handler';
 import healthChekcRouter from './routes/health-check.routes';
 import productsRouter from './routes/products.routes';
+import moviesRouter from './routes/movies.routes';
+import genresRouter from './routes/genres.routes';
 
 const app: Express = express();
 const PORT = 3000;
+const MONGODB_URI =
+  'mongodb+srv://oksanazaharchenko:4bIg0tLnbnA5edNR@cluster0.voxnl81.mongodb.net/NODE_JS_DB?retryWrites=true&w=majority';
```

Comment:
Please do not push sensitive data to GitHub, use .env file for this, for example

Reply:
just wanted to make review easier, sorry, removed sensitive dat